extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -144,6 +146,64 @@ describes.realWin('amp-bind', {
     });
   });
 
+  describe('under dynamic tags', () => {
+    it('should dynamically detect new bindings', () => {
+      const doc = env.win.document;
+      const template = doc.createElement('template');
+      doc.getElementById('parent').appendChild(template);
+      return onBindReady().then(() => {
+        expect(bind.boundElements_.length).to.equal(0);
+        createElementWithBinding('[onePlusOne]="1+1"');
+        return bind.waitForAllMutationsForTesting();
+      }).then(() => {
+        expect(bind.boundElements_.length).to.equal(1);
+      });
+    });
+
+    it('should NOT bind blacklisted attributes', () => {
+      env.sandbox.restore();
```

Comment:
`beforeEach` sets canBind and isResultValid to always return true. In this test I need those methods to use their real implementation. This should probably be moved over to the integration tests once those are merged in.